src/main/java/com/salesforce/emp/connector/LoginHelper.java

```diff
@@ -77,7 +77,7 @@ public void startElement(String uri, String localName, String qName, Attributes
         }
     }
 
-    public static final String COMETD_REPLAY = "/cometd/replay/";
```

Comment:
D'oh.

And so it is revealed why the bug existed in the first place.

Thanks. ;)